src/Components/Web.JS/src/InputFile.ts

```diff
@@ -101,9 +101,9 @@ async function ensureArrayBufferReadyForSharedMemoryInterop(elem: InputElement,
   getFileById(elem, fileId).arrayBuffer = arrayBuffer;
 }
 
-async function readFileData(elem: InputElement, fileId: number, startOffset: number, count: number): Promise<Uint8Array> {
+async function readFileData(elem: InputElement, fileId: number): Promise<Uint8Array> {
   const arrayBuffer = await getArrayBufferFromFileAsync(elem, fileId);
-  return new Uint8Array(arrayBuffer, startOffset, count);
+  return new Uint8Array(arrayBuffer);
```

Comment:
Just want to check: did `InputFile` previously support random access on the file data stream? Were the `startOffset`/`count` used in some useful way that allowed you to fetch just a subset of the file data from the middle, for example?

If so is it possible we're losing a feature here? Not that it sounds like a vital feature but I just want to be sure I understand.

Reply:
> Just want to check: did `InputFile` previously support random access on the file data stream? Were the `startOffset`/`count` used in some useful way that allowed you to fetch just a subset of the file data from the middle, for example?

We had [previously implemented a makeshift streaming implementation](https://github.com/dotnet/aspnetcore/pull/33491/files/e37413beae5f4ddc1ca0bb22f1e43c02bcbecd53#diff-70374d5de5f62a5dc1978ebbe28d2f4a5bbd7c9e76eee2018bfcbd9b2de4d19eR40-R48) to enable `InputFile`. That implementation required `startOffset`/`count` to function appropriately. The previous implementation has been completely revamped to leverage the new streaming interop support and thus `startOffset`/`count` aren't required (as the stream handles the offsets for us as we read through the stream (`Stream.Position`/`chunkSize`). 

> If so is it possible we're losing a feature here? Not that it sounds like a vital feature but I just want to be sure I understand.

Not unless someone was calling into the framework code `InputFileInterop.ReadFileData` directly. This would've been undocumented behavior anyways so I'm not too worried about this.